MobileCenter/MobileCenter/Internals/AppDelegate/MSAppDelegateUtil.h

```diff
@@ -0,0 +1,18 @@
+#if TARGET_OS_OSX
+#import <AppKit/AppKit.h>
+#ifndef MSOriginalAppDelegate
```

Comment:
Thing is it's then not easy to differentiate `MSApplicationDelegate` and `MSAppDelegate`. It's not obvious that the `MSApplicationDelegate` is the original one since it's prefixed by our prefix ^^. We can probably rename `MSApplication` though since there is no custom application.

Reply:
Yes, I agree that it will be hard to distinguish between `MSApplicationDelegate` and `MSAppDelegate`.
I would recommend renaming `MSAppDelegate` to something else. 😄  Something like `MSCustomApplicationDelegate` and `MSApplicationDelegateForwarder`?

Reply:
Renaming MSApplicationDelegateForwarder will be a breaking change because of the `isAppDelegateForwarderEnabled` API. Although the term `AppDelegate` is quite known in th iOS world.

Reply:
I think the choice is either `MSCustomApplicationDelegate` and `MSApplicationDelegate` or `MSCustomAppDelegate` and `MSAppDelegate`. Knowing that we should keep `MSAppDelegateForwarder`. @TroubleMakerBen  what's your thoughts.

Reply:
Naming will now be: `MSApplicationDelegate`, `MSCustomApplicationDelegate` and `MSAppDelegateForwarder`